tests/acceptance/features/apiSharingNg1/sharedByMe.feature

```diff
@@ -3863,3 +3863,806 @@ Feature: resources shared by user
         }
       }
       """
+
+  @env-config
+  Scenario: sharer lists file share with role Secure Viewer after share role is disabled (Personal Space)
```

Comment:
I had seen a similar step. Can we minimise the use of multiple scenarios?

Reply:
I don't think there is specific scenario where share permission role is disabled.